test/e2e/apps/deployment.go

```diff
@@ -40,13 +40,18 @@ import (
 	deploymentutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/replicaset"
-	testutil "k8s.io/kubernetes/test/utils"
+	testutils "k8s.io/kubernetes/test/utils"
 	utilpointer "k8s.io/utils/pointer"
 )
 
 const (
 	dRetryPeriod  = 2 * time.Second
 	dRetryTimeout = 5 * time.Minute
+
+	// Poll is how often to Poll pods, nodes and claims.
+	Poll = 2 * time.Second
+	// PollShortTimeout specifies a timeout for polling.
+	PollShortTimeout = 1 * time.Minute
```

Comment:
does it make sense to lowercase these constants?
are they used in other places?

[1]

Reply:
it does make sense to lowercase them. it was a mistake of my part.